### PR TITLE
http bridge with local cluster in integration tests

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

#67

# Overview

- [ ] I have reviewed the integration framework to simplify a bit the context passing, making it more explicit and removing the extra bits not needed
- [ ] I have setup an initial environment for running Byron cardano node in the integration tests
- [ ] I have connected pieces together and launched an actual cluster + wallet backend server + http bridge all together in the integration tests.
- [ ] I have updated build / tests instructions in the README about how to run the actual integration tests
- [ ] TODO  finalize updating travis.yml to have 'cardano-node-simple' in context (I have a pre-compiled binary in the `test/data/cardano-node-simple` that would do the trick).

# Comments

We do nothing with the cluster (for now) but it should be fully functional and allow us to submit transactions and see them being accepted by nodes.

@piotr-iohk This will conflict with #131 but I am happy to adapt either of the PR with the change. Conceptually, it doesn't really interfere with your addition on #131.

### CODE REVIEW

**Review by piotr-iohk — APPROVED** [review]
nice one :+1:

File: test/integration/Main.hs [hunk 1]
```diff
@@ -43,66 +35,94 @@ main = do
...
+        ] (pure ())
+
+    cardanoHttpBridge port network = Command
+        "cardano-http-bridge"
+        [ "start"
+        , "--port", port
+        , "--template", network
+        ] (threadDelay 5000000)
+
+    cardanoWalletServer serverPort bridgePort network = Command
```
Comment (piotr-iohk) [inline test/integration/Main.hs:88]: Why not `cardanoWalletLauncher` instead `Bridge` and `Server`?

Comment (KtorZ) [reply test/integration/Main.hs:88]: Good point :no_mouth:  ... I have no answers.

**Comment by KtorZ** [comment]
Merging this to get the `cardano-http-bridge` pre-compiled available on master, and making another PR to change the usage of `bridge` + `server` with launcher + update .travis.yaml to actually have cardano-http-bridge in scope.

**Pull request merged**